main.py

```diff
@@ -1,3 +1,23 @@
+import os
+import sys
+from pathlib import Path
+
+PROJECT_DIRECTORY = Path(__file__).parent.absolute()
+
+def addFoldersToPath(directory, recursive=True):
+	'''
+	Directory has to be pathlib format
```

Comment:
could be a single line comment

Reply:
This is a docstring format; you should read the tutorial here: https://www.datacamp.com/community/tutorials/docstrings-python
https://www.pythonforbeginners.com/basics/python-docstrings

This way next time people use the function they can get info from the command line

Reply:
Example: 
![image](https://user-images.githubusercontent.com/33208295/80165758-2b8fcb80-85aa-11ea-9d8e-fa6457747bb6.png)